web_scraper.py

```diff
@@ -0,0 +1,65 @@
+from bs4 import BeautifulSoup
+import requests
+
+def parse_pet(href):
+    html = requests.get(href)
+    html_text = html.text
+    soup = BeautifulSoup(html_text, 'lxml')
+    details = soup.find('ul', class_ = 'petdetails')
+    indents = details.find_all('li')
+    #W poniższych zmiennych zapisze dane, None oznacza brak danych
+    age = None
+    breed = None
+    gender = None
+    weight = None
+    number = None
+    status = None
+    date_in = None
+    date_out = None
+    box = None
+    address_found = None
+    group_link = None
+    group_name = None
+    for indent in indents:
+        if (indent.text.startswith("W typie rasy")):

```

Comment:
What happens if the website changes and is very different then the version now. We should catch all errors thrown by these functions.